docs/develop/03_workshop/04_attester/02_did.md

```diff
@@ -11,64 +11,98 @@ import TabItem from '@theme/TabItem';
 
 :::info KILT DID
 
-There are currently four different key types that a DID supports:
+There are four different key types that a DID supports:
 
-- An _authentication keypair_, used to sign claims and create authenticated credential presentations
+- An _authentication keypair_, used to sign claims and present authenticated credentials 
 - A _key-agreement keypair_, used to encrypt/decrypt messages
 - An _assertion-method keypair_, used to write CTypes and attestations on chain
 - A _capability-delegation keypair_, used to write delegations on chain
 
-Keys can be replaced over time, e.g., if a key is compromised.
+You can replace keys over time, e.g., if a key becomes compromised.
 
 :::
 
-## Account vs DID
+## What's the difference between a DID and an account?
+
+A DID and an account sound quite similar, but there are some differences:
 
-A full DID needs to be registered on the blockchain.
-For that, an account has to submit the DID creation transaction.
-There is always an account that submits the transactions and pays for the fees and the DID that authorized the call.
-Because the DID and the account are not connected, DIDs do not hold any coins.
+- You record both to chain
+- You can have a DID without an account
+- You can have an account without a DID
+- Only an account can pay deposits and fees and start a DID verification process
+- DIDs don't hold any coins
+
+In summary, you register a DID on the blockchain by an account submitting the DID creation transaction and paying the fees.
 
 ## Create a DID
 
-To create a DID we can use the same keyrings that are used to generate accounts.
-For our <span className="label-role attester">Attester</span> we'll need all four types of keys.
-Since three of the key types are used for signatures, we can use the same key for these.
-We'll use the default KILT keyring to generate them.
+As an <span className="label-role attester">Attester</span> needs to interact with the chain, you must create a full DID.
+
+### Generate keypairs
+
+An <span className="label-role attester">Attester</span> needs all four types of key.
+Since three of the key types write and sign to the chain, you can use the same key for them using the default KILT keyring to generate them, which is the same keyring used to generate accounts.
+
+Add the following code to the `attester/generateKeypairs` file.
 
 <TsJsBlock fileName="attester/generateKeypairs">
   {GenerateKeypairs}
 </TsJsBlock>
 
-Once we have created all the necessary keys for a DID we can create the on-chain DID.
-To create a DID we first initialize everything.
-After that, we load the account that we created in the [last section](./01_account.md).
-The account will be used to pay for the DID registration.
-Finally, we create and submit the extrinsic (aka transaction) that will register our DID.
+<!-- TODO: Is this enough? -->
```

Comment:
Does the following description describe enough detail?

Reply:
@ntn-x2 Ahh, but you know the subject ;) A workshop is typically pretty detailed…